x-pack/plugins/integration_assistant/public/common/lib/api_parsers.ts

```diff
@@ -13,13 +13,20 @@ import type { EpmPackageResponse } from './api';
  * TODO: Return the package name from the fleet API: https://github.com/elastic/kibana/issues/185932
  */
 export const getIntegrationNameFromResponse = (response: EpmPackageResponse) => {
-  const ingestPipelineName = response.response?.[0]?.id;
-  if (ingestPipelineName) {
-    const match = ingestPipelineName.match(/^.*-([a-z\d_]+)\..*-([\d\.]+)$/);
-    const integrationName = match?.at(1);
-    const version = match?.at(2);
-    if (integrationName && version) {
-      return `${integrationName}-${version}`;
+  const items = response?.items?.map((item) => item);
+  if (items != null) {
+    for (const item of items) {
```

Comment:
We should be able to simplify it with
```suggestion
    for (const { type, id } of items) {
```

Reply:
Changed it from `for` loop